plutus-metatheory/src/Untyped/CEKWithCost.lagda.md

```diff
@@ -36,7 +42,15 @@ CekM : Set → Set
 CekM = Writer Budget
 
 spend : StepKind → CekM ⊤
-spend st = tell (cekMachineCost st)
+spend st = tell (cekMachineCost (BStep st))
```

Comment:
This isn't attempting to deal with slippage, but that's OK because the only case where it'll make a difference is when you fail due to exceeding your budget, is that correct?  It might be worth adding a comment about that.

Reply:
`plc-agda` does not implement a `restricting` mode. So there is no need to implement slippage.

Reply:
Done